src/btree/bt_vrfy.c

```diff
@@ -13,11 +13,12 @@
  * prettier.
  */
 typedef struct {
-    uint64_t record_total; /* Total record count */
+    uint64_t records_so_far; /* Total record count seen so far */
```

Comment:
Record count seen so far, seems like a better comment as total and count are effectively the same thing here.

Reply:
Pushed in 66ccd6594.